src/components/Button/CircularProgress.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import styles from './CircularProgress.module.css';
+
+const CIRCLE_SIZE = 44;
+const THICKNESS = 3.6;
+
+export const CircularProgress: FC = ({ ...props }) => {
+  return (
+    <span role="progressbar" className={styles.root} {...props}>
+      <svg viewBox={`${CIRCLE_SIZE / 2} ${CIRCLE_SIZE / 2} ${CIRCLE_SIZE} ${CIRCLE_SIZE}`}>
```

Comment:
```suggestion
      <svg role="img" aria-label="読み込み中" viewBox={`${CIRCLE_SIZE / 2} ${CIRCLE_SIZE / 2} ${CIRCLE_SIZE} ${CIRCLE_SIZE}`}>
```

For screen readers, text is required.

Reply:
`progressbar role`

👍